utils/shell-docker.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
No pressure to change all of these but... if you're interested in these scripts running on NixOS, these shebangs should be `#!/usr/bin/env bash`, etc.